contrib/basicauth/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+nginx:
+  build: "nginx"
+  ports:
+    - "5441:5441"
+  links:
+    - registryv2:registryv2
+registryv2:
+  build: "../../"
+  ports:
+    - "5000"
```

Comment:
When I tested out the procedure, it worked as documented.  I'll differ to @dmcgowan for the technical point as everything is based on his test infrastructure.